### PR TITLE
feat: add seed peer back source traffic

### PR DESCRIPTION
**Pull request by jim3ma**
Signed-off-by: Jim Ma <majinjing3@gmail.com>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Review by gaius-qi — APPROVED** [review]
LGTM

File: scheduler/resource/seed_peer.go [hunk 1]
```diff
@@ -137,6 +138,8 @@ func (s *seedPeer) TriggerTask(ctx context.Context, task *Task) (*Peer, *schedul
 			peer.FinishedPieces.Set(uint(piece.PieceInfo.PieceNum))
 			peer.AppendPieceCost(pkgtime.SubNano(int64(piece.EndTime), int64(piece.BeginTime)).Milliseconds())
 			task.StorePiece(piece.PieceInfo)
+
+			metrics.Traffic.WithLabelValues(peer.Tag, peer.Application, metrics.TrafficBackToSourceType).Add(float64(piece.PieceInfo.RangeSize))
```
Comment (bigerous) [inline scheduler/resource/seed_peer.go:142]: 在这里统计不够准确，seedpeer download 万一失败了呢

Comment (jim3ma) [reply scheduler/resource/seed_peer.go:142]: 目前只上报成功的 piece 信息，或者直接失败，所以这里统计没有问题的。

**Review by bigerous — APPROVED** [review]
LGTM

**Pull request merged**